sdk/python/packages/flet-core/src/flet_core/navigation_bar.py

```diff
@@ -61,10 +58,10 @@ def __init__(
         self.selected_icon_content = selected_icon_content
         self.bgcolor = bgcolor
 
-    def _get_control_name(self):
-        return "navigationdestination"
+    def _get_control_name(self) -> str:
```

Comment:
We could add `-> str` only in `Control. _get_control_name`, to avoid having it in all controls.

Reply:
ok, i'll fix that